apiserver/pkg/common/common.go

```diff
@@ -91,3 +92,41 @@ func GetAPIServer(ctx context.Context, dynamicClient dynamic.Interface, external
 	}
 	return api, nil
 }
+
+func GetObjStatus(obj client.Object) string {
```

Comment:
add some comments to let developers know that they should all get object by following this way